javascript/react/src/components/ArqueroDetailsList/renderers/StatsColumnHeader.tsx

```diff
@@ -19,6 +19,7 @@ const pretty: Record<string, string> = {
 const CELL_HEIGHT = 14
 
 const DEFAULT_STATS: StatsColumnType[] = [
+	StatsColumnType.Type,
```

Comment:
What do Type and Invalid indicate here?

Reply:
This is a enum of possible stats. Type shows the data type for the column, Invalid shows the count of invalid values (null, undefined, NaN)